IVAN_MARAKHOVSKII/largest-smallest.js

```diff
@@ -0,0 +1,14 @@
+function largest(){
+    let max = arguments[0];
+    for (let i = 1 ; i < arguments.length ; i++){
+        if (arguments[i] > max ) max = arguments[i]
+    }
+    return max
+}
+function smallest(){
+    let min = arguments[0];
+    for (let i = 1 ; i < arguments.length ; i++){
+        if (arguments[i] < min ) min = arguments[i]
+    }
+    return min
```

Comment:
There are methods like `.largest` and `.smallest` that could be used fir simplicity

Reply:
it's undefined when i'm trying to get arguments.smallest() or arguments.smallest